crates/common/download/src/download.rs

```diff
@@ -1,9 +1,8 @@
 use crate::error::DownloadError;
 use backoff::{future::retry, ExponentialBackoff};
-use nix::{
-    fcntl::{fallocate, FallocateFlags},
-    sys::statvfs,
-};
+#[cfg(target_os = "linux")]
```

Comment:
The changes in this file were made just to make sure the project compile on my non-Linux dev machine. I could move it to a dedicated PR if it's annoying. Left it in this PR itself as the changes were limited to just this file,